hbase-plugins/docs/HBase-batchsink.md

```diff
@@ -1,19 +1,20 @@
 # HBase Batch Sink
 
+
 Description
 -----------
-
 Writes records to a column family in an HBase table with one record field mapping
 to the rowkey, and all other record fields mapping to table column qualifiers.
 This sink differs from the Table sink in that it does not use CDAP datasets, but writes
 to HBase directly.
 
+
 Use Case
 --------
-
 The sink is used whenever you need to write to an HBase table in batch. For example,
 you may want to periodically dump the contents of a relational database into an HBase table.
 
+
 Properties
 ----------
 **tableName:** The name of the table to read from.
```

Comment:
there is an issue here, this should be table to write to.. this seems to be copied from batchsource.